tests/ansi/creal-cimag.c

```diff
@@ -18,6 +18,7 @@
     )
 
 int main() {
+#if !(defined(USE_HOST_LIBC) && defined(__clang__))

```

Comment:
What's up with this? Does clang with glibc produce significantly different results?

Reply:
The commit message has an explanation for this.

Reply:
Ah, my bad.